src/cli/usage.ts

```diff
@@ -1,11 +1,15 @@
 export function createUsage() {
-  console.log('Usage:');
-  console.log('depngn <node-version> [options]');
-  console.log('');
-  console.log('Options:');
-  console.log('-h, --help       output usage information');
-  console.log('-r, --reporter   which reporter for output. options are: terminal (default), json');
-  console.log('');
-  console.log('Example:')
-  console.log('depngn 12.0.0 --reporter=json');
+  const usage = `
+  Usage:
+  depngn <node-version> [options]
+
+  Options:
+  -h, --help      output usage information
+  -r, --reporter  which reporter for output. options are: terminal (default), json
+
+  Example:
+  depngn 12.0.0 --reporter=json
+  `
+
+  console.log(usage);
```

Comment:
As you can see in the image the message now has a kind of left-padding.
if you disagree with that I can revert the change, also, If you prefer using multi-console logs I can revert it too.

![lll-2022-10-02_00-19](https://user-images.githubusercontent.com/7331511/193439326-b7d41195-0a4c-4a1e-91ea-1d477150eefc.png)

Reply:
oohh, i actually kind of like this!

Reply:
the only weird thing that might pop up is if we set up automatic formatting/linting -- it might mistakenly try to "fix" this and add even more padding. we can cross that bridge when we come to it though.